app/src/main/java/com/example/mykotlincalculator21/mvp/view/CalculatorView.kt

```diff
@@ -11,4 +13,25 @@ class CalculatorView(mainActivity: MainActivity, private val binding: ActivityMa
         binding.screenTextExample.setText(number)
     }
 
+    override fun showErrorDivision() {
+        val context = getContext()
+        if (context != null) {
```

Comment:
here you are not using the context. why do you question if it is null?

Reply:
Maybe you can do like this
binding.screenTextExample.setText(context?.getString(R.string.....))

Reply:
It is better like it is now without the use of the context so you can just remove the context

Reply:
Thx for the tip, now its done